lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +234,46 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        links = article_bs.find_all('a', class_="nohover")
+        for link in links:
```

Comment:
this method should extract just one url from bs4 element given, why you try to loop here?

Reply:
I don't know how to do it the other way.Otherwise it collects one article and puts 100 copies of it in a tmp folder.

Reply:
you may for-loop in find_articles method to get each bs4 element with link 

then call_extract_url method for each bs4 element